alpine-rails-nginx/install-nginx.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+NGINX_HEADERS_MORE_VERSION=0.261
+NGINX_VERSION=1.9.5
+
+function verified_curl {
+  url="$1"
+  file="$2"
+  sha256_hash="$3"
+  curl --silent --fail --location "$url" >| "$file" \
+    && echo "$sha256_hash  $file" | sha256sum -cs \
+    && tar xf "$file"
+}
+
+mkdir /usr/local/src
+cd /usr/local/src
+
+verified_curl \
+  "http://nginx.org/download/nginx-$NGINX_VERSION.tar.gz" \
+  "nginx-$NGINX_VERSION.tar.gz" \
+  '48e2787a6b245277e37cb7c5a31b1549a0bbacf288aa4731baacf9eaacdb481b'
```

Comment:
I think we should move the SHA literal to a constant next to the version. For easier change.